docs/src/main/docs/about/01_overview.adoc

```diff
@@ -0,0 +1,103 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
[nit, missing comma]

Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.

Reply:
We actually don't put a comma when there is a single year, see the template at etc/copyright.txt